src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
Are you going to add unit tests? Just wondering because we don't have any on our current dApps but obviously this would be great.

Reply:
That's the plan!

Reply:
+1